2.4-micro/Dockerfile.c10s

```diff
@@ -60,16 +60,16 @@ ENV HTTPD_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/httpd/ \
     HTTPD_DATA_ORIG_PATH=/var/www \
     HTTPD_LOG_PATH=/var/log/httpd
 
-COPY 2.4-micro/s2i/bin/ $STI_SCRIPTS_PATH
-COPY 2.4-micro/root /
-COPY 2.4-micro/core-scripts/usr /usr
+COPY ${HTTPD_VERSION}-micro/s2i/bin/ ${STI_SCRIPTS_PATH}
```

Comment:
It looks like the problem was caught here for RHEL8 https://github.com/sclorg/nginx-container/pull/184. Especially https://github.com/sclorg/nginx-container/commit/7af4049fb857cccb674d00bda444795ddaf9c07a. I still do not see benefit, that we spent on these changes. It looks like Konflux works properly https://gitlab.com/redhat/rhel/containers/mariadb-105/-/merge_requests/198.